src/chrome-launcher.ts

```diff
@@ -304,7 +304,7 @@ class Launcher {
   waitUntilReady() {
     const launcher = this;
 
-    return new Promise((resolve, reject) => {
+    return new Promise<{}>((resolve, reject) => {
```

Comment:
these could probably be better expressed as `Promise<void>` but it's not a big deal :)

Reply:
I was going to, but these existing spots had `{}` so I left it:
https://github.com/GoogleChrome/chrome-launcher/blob/master/test/chrome-launcher-test.ts#L26
https://github.com/GoogleChrome/chrome-launcher/blob/master/src/chrome-launcher.ts#L50

Reply:
heh, yeah those are mistakes IMO, but nbd